apps/web/src/config/index.ts

```diff
@@ -61,7 +61,8 @@ export const IS_TEMPLATE_STORE_ENABLED = isCypress
   : window._env_.IS_TEMPLATE_STORE_ENABLED || process.env.IS_TEMPLATE_STORE_ENABLED || 'false';
 
 export const IS_MULTI_PROVIDER_CONFIGURATION_ENABLED = isCypress
-  ? window._env_.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED ||
+  ? window._cypress.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED ||
```

Comment:
use the flag

Reply:
Those config conditions are becoming more and more complex to understand at first glance, maybe we could do another approach in the near future. (not related to this PR :) )